thirdparty/rr-full/plugins/itempos.pl

```diff
@@ -21,8 +19,7 @@
 # provided, Andrew Case for posting the Registry Decoder code, and Kevin 
 # Moore for writing the shell bag parser for Registry Decoder, as well as 
 # assistance with some parsing.
-#
-# License: GPL v3 
+# 
```

Comment:
Is that license not valid any more?

Reply:
The license appears to have been removed from most files in favor of a license.txt file.  This was removed in 2018 by keydet89 in the the regripper2.8 repo.